rest/route/pod_test.go

```diff
@@ -16,6 +17,12 @@ import (
 )
 
 func TestPostPod(t *testing.T) {
+	env := evergreen.GetEnvironment()
```

Comment:
Instead of setting global environment state, use the `testutil.NewEnvironment` on L84 and set the settings on that one.

Reply:
ph.Run does not have the ecs config when it's only set on testutil. I moved it down there but it has to be a a global env.

Reply:
It seems like it doesn't have to be set on the global `evergreen.GetEnvironment()`, just on the environment passed into the route handler (see my other comment).